src/reason_pprint_ast.ml

```diff
@@ -1177,9 +1179,7 @@ let easyFormatToFormatter f x =
     Easy_format.Pretty.define_styles fauxmatter html_escape html_style;
   let _ = Easy_format.Pretty.to_formatter fauxmatter x in
   let trimmed = strip_trailing_whitespace (Buffer.contents buf) |> String.trim in
-  Format.fprintf f "%s\n" trimmed;
-  pp_print_flush f ()
-
+  Format.fprintf f "%s\n" trimmed
```

Comment:
Is this diff missing something? We've removed the flush call, but then someone else has to invoke the flush now, so where does that happen?